ethcore/sync/src/chain/supplier.rs

```diff
@@ -266,6 +282,10 @@ impl SyncSupplier {
 				added_receipts += receipts_bytes.len();
 				added_headers += 1;
 				if added_receipts > MAX_RECEIPTS_TO_SEND { break; }
+				// Check that the packet won't be oversized
+				if data.len() > payload_soft_limit {
```

Comment:
We already check `MAX_RECEIPTS_TO_SEND` which actually seems to operate on bytes not the receitps (?)

Reply:
Yes you're right, sorry.